packages/amplify-category-function/src/provider-utils/awscloudformation/utils/permissionMapUtils.ts

```diff
@@ -1,5 +1,8 @@
+import { FeatureFlags } from 'amplify-cli-core';
 import _ from 'lodash';
 
+const featureFlagPR5342 = () => FeatureFlags.getBoolean('useAppSyncPermissionsInLambdaPolicies')
```

Comment:
Just name it normally as a variable, no need for GH reference, also FeatureFlags has a <group>.<feature>, I'd suggest to use `appSync`.`generateGraphQLPermissionsForCRUD` since this feature flag is not for `graphqlTransformer`. We can debate the name but this is what it does IMHO, suggest something similar if you have a better in mind.

Reply:
how do you feel about the category being `iamPolicies`? This distinction would be helpful to make clear it's related to how the policies are generated and managed.